lib/vagrant-libvirt/action/start_domain.rb

```diff
@@ -33,12 +33,25 @@ def call(env)
               xml_descr = REXML::Document.new descr
               descr_changed = false
 
+              # additional disk bus
+              config.disks.each {|disk|
+                device = disk[:device]
+                bus = disk[:bus]
+                REXML::XPath.each(xml_descr,'/domain/devices/disk[@device="disk"]/target[@dev="'+device+'"]') {|disk_target|
+                  if disk_target.attributes['bus'] != bus
+                    descr_changed = true
+                    disk_target.attributes['bus'] = bus
+                    disk_target.parent.delete_element("#{disk_target.parent.xpath}/address")
+                  end
+                }
+              }
+
               # disk_bus
-              REXML::XPath.each(xml_descr,'/domain/devices/disk[@device="disk"]/target') {|disk_target|
+              REXML::XPath.each(xml_descr,'/domain/devices/disk[@device="disk"]/target[@dev="vda"]') {|disk_target|
```

Comment:
Nothing, it will skip it. 
The template defines the first disk as vda. So until the user changes it by hand it will be vda. 
Manual changes can't be supported. It yields too many variants.